docs-src/stargate-develop/modules/develop/examples/docapi/library/curl-data-get-path-array-in-document.sh

```diff
@@ -0,0 +1,4 @@
+curl --location -g --request GET '{base_doc_url}{base_doc_api}/{namespace}/collections/{collection}/{readerdocid}/reader/reviews/[1]' \
+--header "X-Cassandra-Token: {auth_token}" \
+--header 'Content-Type: application/json' \
+--data-raw ''
```

Comment:
No need for `--data-raw ''` in any of these curl requests! It can be omitted

Reply:
I think postman adds those to the curl code it generates.

Reply:
fixed. global find in vscode for the win!